Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM circleci/python:3.7.0-stretch-browsers
+FROM circleci/python:3.7.0-stretch
 
 # We need at least g++-8, but stretch comes with g++-6
 # Set up the Debian testing repo, and then install g++ from there...
 RUN sudo bash -c "echo \"deb http://ftp.us.debian.org/debian testing main contrib non-free\" >> /etc/apt/sources.list" \
   && sudo apt-get update \
   && sudo apt-get install node-less cmake build-essential clang-format-6.0 uglifyjs chromium ccache \
+  # for extracting firefox and running chrome, respectively
+  && sudo apt-get install -y bzip2 libgconf-2-4 \
```

Comment:
Can we add these to the previous line so it all runs as part of the same `apt-get` transaction?  (Just for speed).